apps/admin/frontend/src/screens/settings_screen.tsx

```diff
@@ -1,25 +1,42 @@
 import React, { useContext } from 'react';
+import { assert } from '@votingworks/basics';
 import {
   CurrentDateAndTime,
   Prose,
   RebootFromUsbButton,
   RebootToBiosButton,
   SetClockButton,
 } from '@votingworks/ui';
+import { isSystemAdministratorAuth } from '@votingworks/utils';
 
 import { AppContext } from '../contexts/app_context';
 import { NavigationScreen } from '../components/navigation_screen';
 import { FormatUsbButton } from '../components/format_usb_modal';
+import { updateSessionExpiry } from '../api';
 
 export function SettingsScreen(): JSX.Element {
-  const { logger, usbDrive } = useContext(AppContext);
+  const { auth, logger, usbDrive } = useContext(AppContext);
```

Comment:
I know I should probably use the `getAuthStatus` query instead of context to get the auth status. Was just fearing having to update tests 🙈

Will change if it ends up being easy and doesn't delay m17a rc2 image creation